scratchai/__init__.py

```diff
@@ -1,12 +1,18 @@
-from scratchai import trainers
+import os
+if 'DISPLAY' not in os.environ:
+  import matplotlib as mpl
+  mpl.use('Agg')
```

Comment:
This helps matplotlib imports to not cause an error and exit when running on servers (without any displays)